Create ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md (#2514)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-32-2-fix-running-karma-tests-against-ngx-components-for-layered-products-7320.md

---
date: ""

> title: Fix running Karma tests against @c8y/ngx-components library

Reviewer: ```suggestion
title: Fixed issue with Karma tests against @c8y/ngx-components library
```

> title: fix running karma tests against ngx-components for layered products (#7320)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fix running Karma tests against ngx-components for layered products
```

Reviewer 2: ```suggestion
title: Fix running Karma tests against @c8y/ngx-components library
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61312
version: 1020.32.2
---

> fix running karma tests against ngx-components for layered products (#7320)

Author: The Claude AI suggested to use the following body:
```suggestion
Previously, running Karma tests against ngx-components for layered products was not working correctly. With this change, the issue has been resolved by fixing the configuration for running Karma tests against ngx-components. Layered products can now successfully run their Karma tests against ngx-components without encountering any problems.
```

Reviewer 2: ```suggestion
Version 1020.26.0 of the @c8y/ngx-components library introduced an issue which made it impossible for consumers of this library to perform Karma tests against it. This issue has been addressed now.
```
